src/frontend/src/lib/constants/app.constants.ts

```diff
@@ -9,6 +9,8 @@ export const STAGING = MODE === 'staging';
 export const BETA = MODE === 'beta';
 export const PROD = MODE === 'ic';
 
+export const TEST = import.meta.env.TEST === true;
```

Comment:
Should be added to `.env.test` file?

Reply:
And, out of curiosity, why not set it up like the other ENV variables? for example

```typescript
export const MODE = VITE_DFX_NETWORK;
export const LOCAL = MODE === 'local';
export const STAGING = MODE === 'staging';
export const BETA = MODE === 'beta';
export const PROD = MODE === 'ic';
```

Reply:
MODE = DFX_NETWORK comes from vite, from our own configuration plugin
TEST comes from Nodejs or vitest